build.gradle

```diff
@@ -408,19 +408,22 @@ test {
             includeGroups 'cloud', 'bucket'
         } else if (TEST_TYPE == "integration"){
             include "**/*IntegrationTest.class"
-            excludeGroups "cloud", "bucket", "python", "funcotatorValidation"
+            excludeGroups "cloud", "bucket", "python", "funcotatorValidation", "variantcalling"
         } else if (TEST_TYPE == "unit") {
             exclude "**/*IntegrationTest.class"
+            excludeGroups "cloud", "bucket", "python", "funcotatorValidation", "variantcalling"
+        } else if (TEST_TYPE == "variantcalling") {
+            includeGroups "variantcalling"
             excludeGroups "cloud", "bucket", "python", "funcotatorValidation"
         } else if (TEST_TYPE == "spark") {
             includeGroups "spark"
-            excludeGroups "cloud", "bucket", "python", "funcotatorValidation"
+            excludeGroups "cloud", "bucket", "python", "funcotatorValidation", "variantcalling"
         } else if (TEST_TYPE == "python") {
             includeGroups "python"
         } else if (TEST_TYPE == "all"){
             //include everything
         } else {
-            excludeGroups "cloud", "bucket", "python", "funcotatorValidation"
+            excludeGroups "cloud", "bucket", "python", "funcotatorValidation", "variantcalling"
```

Comment:
I think that the `variantcalling` tests should not be excluded in the `else` clause here, since the intention of the else clause is to include all the "normal" (unit/integration) tests, I believe.